packages/amplication-server/src/core/git/git.provider.service.ts

```
@@ -78,15 +81,112 @@ export class GitProviderService {
         id: args.gitRepositoryId
       }
     });
+
     if (isEmpty(gitRepository)) {
       throw new AmplicationError(INVALID_GIT_REPOSITORY_ID);
     }
-    await this.prisma.gitRepository.delete({
+
+    try {
+      await this.prisma.gitRepository.delete({
+        where: {
+          id: args.gitRepositoryId
+        }
+      });
+    } catch (error) {
+      console.log(error.message);
+    }
```

Why do we need that? Why not let the run crash?